V2/tezex/src/components/addLiquidity/index.tsx_

```diff
@@ -0,0 +1,187 @@
+import { FC, useState, useEffect } from "react";
```

Comment:
is this file needed I see `.tsx_` as the extension?

Reply:
no, I'll remove it